doc/classes/XRInterface.xml

```diff
@@ -83,6 +90,13 @@
 			[code]true[/code] if this is the primary interface.
 		</member>
 	</members>
+	<signals>
+		<signal name="play_area_changed">
+			<description>
+				Emitted when the play area is changes.
```

Comment:
```suggestion
				Emitted when the play area is changed.
```

Reply:
@BastiaanOlij As mentioned in the previous comment, this signal could include an additional argument to specify which guardian mode is currently active.

Reply:
@aaronfranke yes it should take reference frame and worldscale into account so if either change it should result in `play_area_changed` to be emitted so you can query an updated play area. 

@m4gr3d once we decide on how we'll introduce a unified way of telling the user which mode is active I fully agree this should be a parameter here.